Satyam1923/Spring-Music-Player
# #fixed: issue No. 458

###Description

A clear and concise description of what the PR does.

This PR does the following:

1. Adds a scroll-to-top button in the footer component to enhance user interaction.

2.Fixes the visibility issue by making the button only appear when the user scrolls down the page.

3.Updates the footer component to include the new feature.

###Related Issues:

Fixes #458

###Changes:

List the detailed changes made in this PR.

1.Added a new feature to show a scroll-to-top button.

2.Refactored the footer component to include the new button.

3.Implemented the functionality to show/hide the button based on the scroll position.

###Testing Instructions:
Detailed instructions on how to test the changes. Include any setup needed and specific test cases.

1.Pull this branch.
2.Run npm install to install dependencies.
3.Run npm start to launch the application.
4.setup the frontend and backend according to it.
5.Verify that the scroll-to-top button appears when scrolling down and hides when scrolling up.
6.Click the button to ensure it scrolls the page to the top smoothly.

###Screenshots (if applicable)
[screen-capture (4).webm](https://github.com/Satyam1923/Spring-Music-Player/assets/142730608/d416bc88-8055-42c1-9552-461c0d51df52)


###Additional Context

This PR is based on the need to improve user interaction by providing an easy way to navigate to the top of the page
